classtest.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
+
+//-------------------------VARIABLES-------------------------
+var timeSlot = {
+  label: 'Times: ',
+  times: ['6am|', '7am|', '8am|', '9am|', '10am|', '11am|', '12pm|', '1pm|', '2pm|', '3pm|', '4pm|', '5pm|', '6pm|', '7pm|'],
+  totalLabel: 'Total|'
+};
+var pikePlace = new CookieStore ('First and Pike:', 23, 65, 6.3);
+// console.log(pikePlace.displayCookies());
+var seaTac = new CookieStore ('SeaTac:', 3, 24, 1.2);
+// console.log(seaTac.displayCookies());
+var seattleCenter = new CookieStore('Seattle Center:', 11, 38, 3.7);
+// console.log(seattleCenter.displayCookies());
+var capHill = new CookieStore('Capitol Hill:', 20, 38, 2.3);
+// console.log(capHill.displayCookies());
+var alki = new CookieStore('Alki:', 2, 16, 4.6);
+// console.log(alki.displayCookies());
+var tableEl = document.createElement('table');
+var stores = [pikePlace, seaTac, seattleCenter, capHill, alki];
+var total = 0;
+//----------------------For Loop to run through the times------------
+
+var rowOneEl = document.createElement('tr');
+var anything = document.createElement('th');
+anything.textContent = timeSlot.label;
+rowOneEl.appendChild(anything);
+
+for (var k = 0; k < timeSlot.times.length ; k++){
+  var timeLabelEl = document.createElement('th');
+  var currentTime = timeSlot.times;
+  timeLabelEl.textContent = currentTime[k];
+  rowOneEl.appendChild(timeLabelEl);
+};
+var totalHead = document.createElement('th');
```

Comment:
If you use the El suffix for some of your variables, you should be consistent about using it for all of them, otherwise it's more confusing than not using the suffix at all.